docs/scratch.rst

```diff
@@ -0,0 +1,268 @@
+
+Installing
+~~~~~~~~~~
+
+``pip install urllib3`` or fetch the latest source from
+`github.com/shazow/urllib3 <https://github.com/shazow/urllib3>`_.
+
+Usage
+~~~~~
+
+.. doctest ::
+
+    >>> import urllib3
+    >>> http = urllib3.PoolManager()
+    >>> r = http.request('GET', 'http://example.com/')
+    >>> r.status
+    200
+    >>> r.headers['server']
+    'ECS (iad/182A)'
+    >>> 'data: ' + r.data
+    'data: ...'
+
+
+**By default, urllib3 does not verify your HTTPS requests**.
+You'll need to supply a root certificate bundle, or use `certifi
+<https://certifi.io/>`_
+
+.. doctest ::
+
+    >>> import urllib3, certifi
+    >>> http = urllib3.PoolManager(cert_reqs='CERT_REQUIRED', ca_certs=certifi.where())
+    >>> r = http.request('GET', 'https://insecure.com/')
+    Traceback (most recent call last):
+      ...
+    SSLError: hostname 'insecure.com' doesn't match 'svn.nmap.org'
+
+For more on making secure SSL/TLS HTTPS requests, read the :ref:`Security
+section <security>`.
+
+
+urllib3's responses respect the :mod:`io` framework from Python's
```

Comment:
Please totally ignore this file, as it's just a holding place for old docs as I shift things around.